test-monitor/nitestmonitor.yml

```diff
@@ -0,0 +1,1277 @@
+swagger: '2.0'
+info:
+  version: '1'
+  title: Test Monitor Web Service
+  description: Test Monitor HTTP API
+  contact:
+    name: National Instruments
+    url: 'https://www.ni.com/systemlink'
+    email: support@ni.com
+basePath: /nitestmonitor
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  basicAuth:
+    type: basic
+  cookieAuth:
+    type: apiKey
+    in: header
+    name: Cookie
+security:
+  - basicAuth: []
+  - cookieAuth: []
+x-ni-routing-key: Skyline.TestMonitor
+definitions:
+  Error:
```

Comment:
Jenna suggested some changes to my Error definition, so you should probably try to incorporate those, as these should be pretty similar. At some point in the future we could look into single-sourcing the error type, so we don't all have a copy of it in our docs. I don't think it needs to happen now, given that we're trying to release.